resources/deb/etc/init.d/openhab2

```diff
@@ -104,7 +125,7 @@ case "$1" in
             then
                 log_end_msg 0
             else
-               log_end_msg 1
+                log_end_msg 1
```

Comment:
The extra space was to bring the line to the same alignment used above.  4 spaces?